plugins/inventory/test/lxd.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Not sure what this is supposed to be, but that clearly does not belong in this directory.

Integration tests belong in `tests/interation/targets/inventory_lxd`, the same way other inventory integration tests are sturctured.

Reply:
This is an example of a working configuration. This example is not necessary and I will transfer it to EXAMPLES.

Reply:
Moved to 'tests/integration/targets/inventory_lxd/tasks/'

Reply:
It's not a list of Ansible tasks, so it doesn't belong there either. Please look at the other integration test directories for inventory plugins to see how the tests there work.